content/intro/licencas.md

```diff
@@ -2,46 +2,55 @@
 
 ## Software Livre x Software de Código Aberto
 
-Uma licença que já falamos anteriormente é a licença GNU GPL (que atualmente está em sua versão 3.0), caracterizada por ser uma licença de software livre e garante para o usuário às quatro liberdades do software livre que foram comentadas anteriormente. Para garantir total liberdade de uso existe o conceito de "copyleft", do qual um software é totalmente livre de direitos autorais sobre o software, garantindo domínio público para seu uso, mas, com direitos quanto à propriedade intelectual do portador do software.
+Uma licença que já falamos anteriormente é a licença GNU GPL (que atualmente está em sua versão 3.0), caracterizada por ser uma licença de software livre ou de código aberto (FOSS - Free and Open Source Software - um acrônimo agnóstico para tratar do que for comum às duas vertentes) e garante para o usuário às "quatro liberdades do software" que foram comentadas anteriormente, assim com a "Open Source Definition", derivada indireta das 4 liberdades.
+
+Para manter as liberdades de uso, em produtos derivados, existe o conceito de "copyleft", que é o apelido de uma característica em licenças, como a GNU GPL, que obriga a aplicação da mesma licença em softwares derivados. O Copyleft pode ser diferenciado como "fraco" ou "forte". Copyleft fraco é o que permite o uso do FOSS como parte (em geral lib ou plugin) de um software fechado ou de licença livre, porém incompativel. Esse conceito existe também na coleção de licenças Criative Commons, como "Share Alike".
+
+Licenças FOSS sem Copyleft são cjaadas de "permisivas". Dentre elas temos a MIT e a BSD, que permitem que o software seja diretamante usado por qualquer outro sob qualquer licença. Permite até mesmo o reuso do código sob outra licença, até mesmo fechada.
+
+Software sob licenças FOSS **não** são o mesmo que softwares em "domínio público". As licenças se apoiam na existência do Copyright, mesmo que subvertendo a sua prática padrão e são uma clara normatização do uso, enquanto obras em domínio público são protegidas apenas para manutenção da autoria. (Você não pode dizer que criou a Mona Lisa.)
 
-Uma licença de código aberto garante que o código-fonte de um software seja aberto, mas, permite custos para modificações, manutenção, entre outros serviços que podem ser prestados sob o uso do software em si, atrelados à algum determinado produto. 
+Uma licenças FOSS exigem que o código-fonte de um software esteja disponível e apesar da confusão comum, não impedem a venda ou cobrança para modificações, manutenção, entre outros serviços que podem ser prestados sob o uso do software em si, ou atrelados à algum determinado produto. Uma licença que proibe isso é a CC-by-nc (Creative Commons - Non Comercial). Essa licença não é considerada FOSS (FSF e OSI concordam), porque restringe a liberdade 0 do software livre e as liberdades 2 e 3 não restringem o "como compartilhar".
```

Comment:
Sobrou o "Uma" no singular, no início do parágrafo. Sugestão: "Uma licenças" -> "Licenças"